### ISSUE TITLE
Error in WindowImplX11.cpp

### ISSUE THREAD

**Issue by KrystianPotrykus** [SFML/SFML#1951]
Thanks for raising your issue here! 🙂

Before you submit the issue however, we'd like you to consider the follow points.

* We like to use the issue tracker for confirmed issues.
* If you're stuck with SFML, please use [the forum](https://en.sfml-dev.org/forums/index.php#c3) to get help.

----

## Subject of the issue

there is an error in the constructor:
WindowImplX11::WindowImplX11(VideoMode mode, const String& title, unsigned long style, const ContextSettings& settings) 
 when calculating the window position:
    if(m_fullscreen)
    {
        windowPosition = getPrimaryMonitorPosition();
    }
    else
    {
        windowPosition.x = (DisplayWidth(m_display, m_screen)  - mode.width) / 2;
        windowPosition.y = (DisplayWidth(m_display, m_screen)  - mode.height) / 2;
    }

shouldn't there be:

    if(m_fullscreen)
    {
        windowPosition = getPrimaryMonitorPosition();
    }
    else
    {
        windowPosition.x = (DisplayWidth(m_display, m_screen)  - mode.width) / 2;
        windowPosition.y = (DisplayHeight(m_display, m_screen)  - mode.height) / 2;
    }

**Comment by Bromeon** [comment]
Thanks for the report, and looking into a code! I created a PR (see link above this comment), would you have the time to check if that fixes your problem?

You can check out the PR with [these instructions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally#modifying-an-inactive-pull-request-locally):
```bash
# navigate to your SFML clone
git fetch origin pull/1953/head:x11-fix
git checkout x11-fix
```
And then rebuild SFML.

**Comment by eXpl0it3r** [comment]
Unable to notice a difference while testing, but the fix seems "correct".
Wondering whether this piece of code is actually still needed.

Fixed with #1953
